Sources/sentry/include/SwiftFatalErrorMessage.h

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+#ifndef sentry_native_include_Sentry_SwiftFatalErrorMessage_hh
+#define sentry_native_include_Sentry_SwiftFatalErrorMessage_hh
+
+#include <sentry.h>
+
+#ifdef _WIN32
+#include <atomic>
+
+inline void *getFatalErrorMessageHandle() {
+  HMODULE hSwiftCore = GetModuleHandleA("swiftCore.dll");
+  if (!hSwiftCore) { return nullptr; }
+  auto pGetBuf = reinterpret_cast<std::atomic<const char *> *(*)()>(
+      GetProcAddress(hSwiftCore, "swift_getFatalErrorMessageBuffer"));
+  CloseHandle(hSwiftCore);

```

Comment:
@hjyamauchi , this `CloseHandle` will unload the dll if `GetModuleHandleA` loaded it, which will invalidate the function and buffer pointers. We should either leak the handle (with a comment), or load it once and store it statically (needs a .c source file).